### ISSUE TITLE
Conditionals inside slots not evaluated properly

### ISSUE THREAD

**Issue by filipweidemann** [wrabit/django-cotton#88]
Hey there, we are currently running into what I assume is a small bug when using slots.

Whenever you pass markup with conditionals to a component slot, the conditionals are not evaluated but are rather completely ignored.

Assume the following component "form-field"…
```
<div class="some-tailwind-classes">
  {{ slot }}
</div>
```

 ...and another component using it like this:
```
<c-form-field>
  <label>some label</label>
  <select>
    <option value="1" {% if my_obj.selection == 1 %} selected {% endif %}>Value 1</option>
    <option value="2" {% if my_obj.selection == 2 %} selected {% endif %}>Value 2</option>
  </select>
</c-form-field>
```

The HTML output then has `selected` on both `<option>` elements.

**Comment by wrabit** [comment]
Resolved in v0.9.30
- https://github.com/wrabit/django-cotton/releases/tag/v0.9.30

**Comment by filipweidemann** [comment]
Wow. This was fast @wrabit. Thank you!

OT: the more I use cotton, the more I love it. :)
